katello/katello.spec

```diff
@@ -26,6 +26,10 @@ Source9:    qpid-core-dump
 Source10:   katello-clean-empty-puppet-environments
 Source11:   katello-change-hostname
 Source12:   katello-repository-publish-check
+Source13:   utilities/restore.rb
+Source14:   utilities/backup.rb
+Source15:   utilities/hostname-change.rb
+Source16:   utilities/helper.rb
```

Comment:
@johnpmitsch  I'm afraid it is not possible to have resources in subdirectories [1]. It needs to go to the root dir or in a tarball. 

What were the reasons originally to put these scripts here and not in the katello repo?
Was there any agreement on plan to move these utilities to `foreman-maintain`?

[1] http://rikers.org/rpmbook/node68.html#SECTION03526100000000000000

Reply:
@mbacovsky The main reason of this refactoring is to make it easy to change our naming conventions for backporting these scripts to satellite. But this is also going to make the scripts more testable and is the first step to moving them to foreman-maintain.

I'm not sure what do do about the spec file and handling the subdirectory, if anyone who is more experienced with rpm spec files can help me out, that would be appreciated